lua/alex/remap.lua

```
@@ -158,3 +158,16 @@ end, { desc = "Document Functions only" })
 vim.keymap.set("n", "<leader>ds", function()
 	require("telescope.builtin").lsp_document_symbols()
 end, { desc = "Document Symbols (all types)" })
+
+-- Floaterm keymaps
+vim.keymap.set("n", "<F7>", "<cmd>FloatermNew<CR>", { silent = true, desc = "New floaterm" })
```

Luacheck (1.2.0)

[warning] accessing undefined variable 'vim'

(W113)